libraries/AP_HAL_Linux/SPIDevice.cpp

```diff
@@ -101,8 +101,8 @@ SPIDesc SPIDeviceManager::_device[] = {
 };
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_POCKET
 SPIDesc SPIDeviceManager::_device[] = {
-    SPIDesc("mpu9250",    1, 0, SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 11*MHZ),
-    SPIDesc("ms5611",     1, 1, SPI_MODE_3, 8, SPI_CS_KERNEL,  10*MHZ,10*MHZ),
+    SPIDesc("mpu9250",    1, 1, SPI_MODE_3, 8, SPI_CS_KERNEL,  1*MHZ, 11*MHZ),
+    SPIDesc("bmp280",     1, 0, SPI_MODE_0, 8, SPI_CS_KERNEL,  1*MHZ,10*MHZ),
```

Comment:
I doubt the bmp280 needs different high and low bus speeds

Reply:
tested with 10 - 10  and its ok